### PR TITLE
Reword the "Where to define messages" style docs

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
LGTM, thanks

File: akka-docs/src/main/paradox/typed/style-guide.md [hunk 1]
```diff
@@ -235,35 +235,35 @@ Java
...
 Scala
 :  @@snip [StyleGuideDocExamples.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/StyleGuideDocExamples.scala) { #messages }
 
 Java
 :  @@snip [StyleGuideDocExamples.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StyleGuideDocExamples.java) { #messages }
 
-Sometimes several actors share the same messages, because they have a tight coupling and using message adapters
-would introduce to much boilerplate and duplication. If there is no "natural home" for such messages they can be
-be defined in a separate @scala[`object`]@java[`interface`] to give them a naming scope.
+However, if several actors share the same messages, because they have a tight coupling and using message adapters
```
Comment (jrudolph) [inline akka-docs/src/main/paradox/typed/style-guide.md]: Maybe that whole paragraph can be stripped down to

```markdown
If several actors share the same message protocol, it's recommended to define 
those messages in a separate @scala[`object`]@java[`interface`] for that protocol.
```

**Review by jrudolph — APPROVED** [review]
LGTM with a small suggestion

**Review by jrudolph — APPROVED** [review]
LGTM

**Pull request merged**